charts/node/values.yaml

```diff
@@ -674,6 +674,10 @@ jaegerAgent:
 # -- Annotations to add to the Pod
 podAnnotations: {}
 
+# dnsPolicy can be set to 'ClusterFirst', 'Default', 'None', or 'ClusterFirstWithHostNet'
```

Comment:
We're using a specific format for annotating values. Check out the rest of the file. All the comments start with `# --`. This is a special syntax used by `helm-docs` tool that we use for autogenerating README file based on the annotated values.
Make sure to render an updated README.md by running `helm-docs` locally. Check out our [contributing guide](https://github.com/paritytech/helm-charts/blob/main/CONTRIBUTING.md) for the details.

Reply:
My bad, already pushed a commit to fix it.